calico_containers/tests/st/docker_host.py

```diff
@@ -15,16 +15,21 @@ def __init__(self, name):
         pwd = sh.pwd().stdout.rstrip()
         docker.run("--privileged", "-v", pwd+":/code", "--name", self.name, "-tid", "jpetazzo/dind")
```

Comment:
Just a followup; I don't plan on making this change in this PR. If there's continued disagreement about it, we can go ahead an make an issue for discussion, and I'll make a separate branch for implementing the agreed conclusion.